docs/src/main/asciidoc/security-oidc-bearer-token-authentication-tutorial.adoc

```diff
@@ -256,15 +255,16 @@ For more information, see the xref:security-keycloak-admin-client.adoc[Quarkus K
 include::{includes}/devtools/dev.adoc[]
 ====
 * xref:security-openid-connect-dev-services.adoc[Dev Services for Keycloak] will start a Keycloak container and import a `quarkus-realm.json`.
-. Open a xref:dev-ui.adoc[Dev UI], which you can find at http://localhost:8080/q/dev-ui[/q/dev-ui], then click a `Provider: Keycloak` link in an `OpenID Connect` `Dev UI` card.
+. Open a xref:dev-ui.adoc[Dev UI], which you can find at http://localhost:8080/q/dev-ui[/q/dev-ui].
+Then, in an OpenID Connect `Dev UI` card, click the `Keycloak provider` link .
 . When prompted to log in to a `Single Page Application` provided by `OpenID Connect Dev UI`, do the following steps:
 
  * Log in as `alice` (password: `alice`), who has a `user` role.
- ** Accessing `/api/admin` returns `403`.
- ** Accessing `/api/users/me` returns `200`.
- * Log out and log in as `admin` (password: `admin`), who has both `admin` and `user` roles.
- ** Accessing `/api/admin` returns `200`.
- ** Accessing `/api/users/me` returns `200`.
+ ** Accessing `/api/admin` returns a `403` status code.
+ ** Accessing `/api/users/me` returns a `200` status code.
+ * Log out and log back in as `admin` (password: `admin`), who has both `admin` and `user` roles.
```

Comment:
```suggestion
 * Log out and log back in as `admin` (password: `admin`), with both `admin` and `user` roles.
```

Reply:
@MichalMaler: Many thanks for your suggestions. I implemented your earlier comments, however, for this one, I would prefer to keep as it is because similar language to describe the access privileges of other users is used i.e. the users "have" certain roles. I hope this is okay.

Reply:
IMHO `with` does not quite work, one does not login with some roles, someone logs in, and the system checks what roles this user has